system-tests/projects/webpack-preprocessor-ts-loader-compiler-options/cypress/e2e/failing.cy.ts

```diff
@@ -27,8 +27,8 @@ context('validation errors', function () {
   })
 
   verify('validation error', this, {
-    line: 26,
-    column: 8,
+    line: 19,
+    isPreprocessorWithTypescript: true,
```

Comment:
this is needed here since we are having stack codeframe/stacktraces now transpiled from typescript 4 to 5, which is slightly different than what is expected in the javascript context